apps/src/bin/quiche-server.rs

```diff
@@ -61,22 +61,19 @@ fn main() {
     let args = ServerArgs::with_docopt(&docopt);
 
     // Setup the event loop.
-    let poll = mio::Poll::new().unwrap();
+    let mut poll = mio::Poll::new().unwrap();
     let mut events = mio::Events::with_capacity(1024);
 
     // Create the UDP listening socket, and register it with the event loop.
     let socket = net::UdpSocket::bind(args.listen).unwrap();
+    socket.set_nonblocking(true).unwrap();
 
     info!("listening on {:}", socket.local_addr().unwrap());
 
-    let socket = mio::net::UdpSocket::from_socket(socket).unwrap();
-    poll.register(
-        &socket,
-        mio::Token(0),
-        mio::Ready::readable(),
-        mio::PollOpt::edge(),
-    )
-    .unwrap();
+    let mut socket = mio::net::UdpSocket::from_std(socket);
```

Comment:
I think we can merge L68 (calling `bind()`) and this line into calling `mio::net::UdpSocket::bind()` once.

Reply:
Sounds good, thanks!
The result of `mio::net::UdpSocket::bind()` seems a non-blocking socket by default, so we can eliminate the `set_nonblocking` call too.

Reply:
Fixed in https://github.com/cloudflare/quiche/pull/1202/commits/935e8d6ac79340a4172da931cd90f66ec963ea67
Note that I also updated the UDP socket creation code merged in the previous PR (#1200).